0.6.1/docker-entrypoint.sh

```diff
@@ -61,4 +61,9 @@ elif vault --help "$1" 2>&1 | grep -q "vault $1"; then
     set -- vault "$@"
 fi
 
+# If we are running Vault, make sure it executes as the proper user.
+if [ "$1" = 'vault' ]; then
```

Comment:
You're right, and a bind mount will reset the owner as root, but we're also mounting sub directories not /vault directly (from memory?). But to be safe, if there is no reasons for it, I should remove the chown.

As for chown to vault from the entrypoint, does vault has data there that needs modifications, or just read access?

Consul needs to write, but does vault?

In my case I only have a /vault/config/config.json, what are the other possibilities?